### PR TITLE
Add cards in bring event to conclusion

### PR DESCRIPTION
**Pull request by signed**

### CODE REVIEW

File: stacks.js [hunk 1]
```diff
@@ -250,28 +256,36 @@ class BringEventToConclusion extends React.Component {
...
+      <div style={containerStyle}>
+        <EventCard key='event-card-front' eventCardId={this.props.number} side='front' name={this.props.name}/>
+        <div style={choiceStyle}>
+          <button key="a" type="button" onClick={this.selectA} className={this.state.selected === "a" ? "selected" : ""}>A</button>
+          <button key="b" type="button" onClick={this.selectB} className={this.state.selected === "b" ? "selected" : ""}>B</button>
+        </div>
+        {this.state.selected && [
+          <EventCard key='event-card-back' eventCardId={this.props.number} side='back' name={this.props.name}/>,
+          <div style={resolutionStyle}>
+            {this.props.children}
```
Comment (kosta) [inline stacks.js:283]: TIL

**Comment by kosta** [comment]
Nice! 

Very good use of React with props.children, I would not have though of that but that makes perfect sense and tickles my brain cells in good ways. I guess this is a major part in writing re-usable components.

Also: Good work with making this flexboxes. I _reallty_ need to brush up my CSS!

Feature Request: Please add some padding so the flex boxes dont look so cramped.

File: stacks.js (hunk 1 above)
Comment (signed) [reply stacks.js:283]: This was new to me too, but I like it.

**Comment by signed** [comment]
Another react concept that I find interesting to be DRY are [higher order components](https://reactjs.org/docs/higher-order-components.html).
Not sure if I can apply them here, but good to know they exist.

**Comment by kosta** [comment]
Wrap, wrap, wrap! ;)

**Comment by signed** [comment]
padding added

**Pull request merged**